packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -425,26 +403,25 @@ test.serial('writeGranulesFromMessage() returns undefined if message has empty g
   t.is(actual, undefined);
 });
 
-test.serial('writeGranulesFromMessage() saves granule records to DynamoDB/PostgreSQL/Elasticsearch/SNS if PostgreSQL write is enabled', async (t) => {
+test.serial('writeGranulesFromMessage() saves granule records to PostgreSQL/Elasticsearch/SNS if PostgreSQL write is enabled', async (t) => {
```

Comment:
It looks like we removed the check so PostgreSQL writes might always be enabled? Could maybe rename this test.

Reply:
835f039b43